App.tsx

```diff
@@ -140,6 +141,8 @@ const asyncStoragePersister = createAsyncStoragePersister({
   storage: AsyncStorage,
 });
 
+clearUploadCache();
```

Comment:
Is this a TODO? In the final version would we, on startup, read this cache and re-try any uploads that had not succeeded?